### ISSUE TITLE
Remove twisted.protocols.gps

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8787]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8787|
|Type|enhancement|
|Created|2016-08-25 18:21:56Z|
|Branch|https://github.com/twisted/twisted/tree/8787-byebye-oldgps|

It's deprecated, untested, not good, and is replaced by Positioning.

<details><summary>Searchable metadata</summary>

```
trac-id__8787 8787
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__8787_byebye_oldgps 8787-byebye-oldgps
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1472149316866612 1472149316866612
changetime__1473355962671934 1473355962671934
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

PR: https://github.com/twisted/twisted/pull/526

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc set owner to @hawkowl|
|-|-|

https://github.com/twisted/twisted/pull/526#issuecomment-242631285

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set status to `closed`|
|-|-|

In changeset e6c0d8d381ae32b71bed05ca53f72f189386af25
```
#!CommitTicketReference repository="" revision="e6c0d8d381ae32b71bed05ca53f72f189386af25"
Merge 8787-byebye-oldgps: Remove twisted.protocols.gps

Author: hawkowl
Reviewer: rodrigc
Fixes: #8787
```
